src/templates/options.js

```diff
@@ -19,3 +19,9 @@ for (const [key, value] of Object.entries(options)) {
     }
 }
 %>
+
+/* <% if (options.langDir) { %> */
+export const ASYNC_LOCALES = {
+  <%= options.locales.map(l => `'${l.code}': () => import('${options.langDir}/${l.file || l.iso}' /* webpackChunkName: "lang-${l.code}" */)`).join(',\n  ') %>
```

Comment:
The `|| l.iso` part seems new and I'm not sure it makes sense. `iso` value doesn't have a file extension so importing it would likely not work.

Reply:
Resovler (cjs, webpack, vite) does rersolve extension for us. It is convention over configuration. But agree have to move it to new PR later.

Reply:
Also agree. And if we want that, we should also document it.

How is it resolved automatically though? Based on content? How would it know if it's a json or yaml or js?